lib/Condition.ts

```diff
@@ -169,22 +174,22 @@ Condition.prototype.parenthesis = Condition.prototype.group = function (this: Co
 	this.settings.conditions.push(value.settings.conditions);
 	return this;
 };
-Condition.prototype.or = function(this: Condition): Condition {
+Condition.prototype.or = function (this: Condition): Condition {
```

Comment:
Same thing here about linting. Would be great if we could define this in the linter to ensure uniform style throughout.